### DISCUSSION TITLE
How to expose some endPoints to public sames as / endpoint?

### DISCUSSION THREAD

**Discussion by richard457** [serverpod/serverpod#61] [Q&A] [answered]
for some use cases, you may want to expose the API to public usage not only the flutter app, but I can see this is done already in http://localhost:8080/  endpoint, how can I do something similar on an endpoint for example http://localhost:8080/users

**Comment by vlidholt** [comment, marked as answer]
Hi!

If you need a second endpoint, you can just create a new file in the endpoint directory. Make sure that the class is named `UsersEndoint` and extends `Endpoint`. The endpoints you create are accessible outside of Flutter (it's just a JSON API).

If you want to create custom JSON structures or HTML pages, this is what the `serverpod_relic` package is for. However, it is currently a bit experimental and documentation isn't great yet. The plan is to merge Relic with Serverpod in a future update when Relic is production ready.

Hope it helps and cool that you are checking out Serverpod!
